exchanges/gateio/gateio.go

```diff
@@ -168,6 +167,23 @@ var (
 	errInvalidTextValue                 = errors.New("invalid text value, requires prefix `t-`")
 )
 
+// validTimesInForces holds a list of supported time-in-force values and corresponding string representations.
+var validTimesInForces = []struct {
```

Comment:
🚧 Can be either `validTimesInForce` or `validTimesInForce` but not both at the same time. `validTimesInForce` is grammatically correct. Alternative that would avoid grammar police and be more searchable: `validTimeInForceValues`

Carry over from [this original comment](https://github.com/thrasher-corp/gocryptotrader/pull/1382/files/3d806e5f6aa1f6d9d27873f654b876f38b63f962#r2044011180)

```suggestion
// validTimesInForce holds a list of supported time-in-force values and corresponding string representations.
var validTimesInForce = []struct {
```

Reply:
🚧 I might be missing something, but ... why isn't this just a map?
Isn't everything we're doing harder because it's a slice?

Reply:
We did a benchmark test and found out looing over a struct and matching the result is much faster than using a map interface